crates/pathfinder/src/consensus/inner/consensus_task.rs

```diff
@@ -160,7 +159,7 @@ pub fn spawn(
                                  {round}",
                             );
 
-                            let fake_proposals_storage = fake_proposals_storage.clone();
+                            let fake_proposals_storage = storage.clone();
```

Comment:
This appears to work, but @CHr15F0x has been complaining that making proposals from the main DB is problematic (AFAIK because the state diffs have to be stored only _after_ the proposal is accepted by consensus, and computing them before that, i.e. here, interferes with that). Has that been solved?

Reply:
+1, i don't remember exactly but unless we have a decently working __real__ proposal creation algo I'd keep this PR/issue parked. Currently using the prod db as a scratchpad for fake proposal creation seems fishy.